openfermioncirq/variational/ansatzes/default_initial_params_test.py

```diff
@@ -20,25 +20,71 @@
 
 from openfermioncirq import (
         HamiltonianObjective,
+        LowRankTrotterAnsatz,
         SplitOperatorTrotterAnsatz,
         SwapNetworkTrotterAnsatz,
         VariationalStudy,
         prepare_gaussian_state,
         simulate_trotter)
-from openfermioncirq.trotter import LINEAR_SWAP_NETWORK, SPLIT_OPERATOR
+from openfermioncirq.trotter import (
+        LINEAR_SWAP_NETWORK, LOW_RANK, LowRankTrotterAlgorithm, SPLIT_OPERATOR)
 
 
-# Construct a Hamiltonian for testing
-test_hamiltonian = random_diagonal_coulomb_hamiltonian(4, real=True, seed=47141)
+def load_molecular_hamiltonian(
```

Comment:
Why is this function showing up all over the place!!??

Reply:
@kevinsung what's the deal with this? Didn't we agree this should go in OpenFermion?

Reply:
Done.